.idea/codeStyles/Project.xml

```diff
@@ -24,7 +24,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
These changes should appear on master ASAP because it's very annoying when you do something with git

Reply:
please see https://github.com/owncloud/android/pull/2960